float32[] globwaypointsx
float32[] globwaypointsy
